apps/worker/app/routers/runs.py

```
@@ -0,0 +1,186 @@
+        # Emit progress event via Socket.IO
+        await emit_progress(
+            run_id, {"status": "initialized", "session_url": session_url}
+        )
+
+        return CreateRunResponse(run_id=run_id, session_url=session_url)
+
+    except HTTPException:
+        # Re-raise HTTPExceptions as-is (they have the correct status code)
+        raise
+
+    except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)